runtime/js-compute-runtime/builtins/server-info.cpp

```diff
@@ -0,0 +1,101 @@
+#include "builtins/server-info.h"
+#include "core/geo_ip.h"
+#include "host_interface/host_api.h"
+#include "js/JSON.h"
+#include "openssl/evp.h"
+#include <arpa/inet.h>
+
+namespace builtins {
+
+namespace {
+
+JSString *address(JSObject *obj) {
+  JS::Value val = JS::GetReservedSlot(obj, static_cast<uint32_t>(ServerInfo::Slots::Address));
+  return val.isString() ? val.toString() : nullptr;
+}
```

Comment:
This seems like it doesn't need to be a dedicated function, and could just as well be inlined into `address_get` since it has a single caller.

Reply:
Can do yes, I was keeping it similar to the implementation in Client info, happy to refactor it